### PR TITLE
Sugestion: add docker option to run #1

### PR DESCRIPTION
**Pull request by martinsrso**

### CODE REVIEW

File: .directory [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Dolphin]
```
Comment (rodrigooler) [inline .directory]: What is .directory?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```
Comment (rodrigooler) [inline .dockerignore:2]: good!

**Review by rodrigooler — CHANGES_REQUESTED** [review]
Good, but the package.json scripts were missing, to make it easier to use docker for the final dev. Could you ship to close the pull request?

File: .directory (hunk 1 above)
Comment (martinsrso) [reply .directory]: .directory is file default in dophin on kde. =/

Comment (rodrigooler) [reply .directory]: remove this please.

**Review by martinsrso — COMMENTED** [review]
Trade fairs/

File: package.json [hunk 3]
```diff
@@ -38,11 +40,33 @@
     "type": "git",
     "url": "git+https://github.com/rodrigooler/apollo-server-beta.git"
   },
+  "config": {
+    "imageRepo": "martinsrso/apollo-server",
```
Comment (rodrigooler) [inline package.json]: change this

File: package.json [hunk 4]
```diff
@@ -38,11 +40,33 @@
...
+    "imageName": "festive_poincare",
```
Comment (rodrigooler) [inline package.json]: change this.

**Review by rodrigooler — APPROVED** [review]
Good!

**Pull request merged**